mirrord/protocol/src/codec.rs

```diff
@@ -141,6 +141,7 @@ pub enum DaemonMessage {
     /// NOTE: can remove `RemoteResult` when we break protocol compatibility.
     GetEnvVarsResponse(RemoteResult<HashMap<String, String>>),
     GetAddrInfoResponse(GetAddrInfoResponse),
+    /// Pause is deprecated but we don't want to break protocol
```

Comment:
Maybe add [this](https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute)?

Reply:
We still want agent to respond with nice log until we break protocol, so I don't think this is helpful.

Reply:
Okok